libs/accounts/src/lib/accounts-table.spec.tsx

```diff
@@ -24,47 +25,104 @@ const singleRow: AccountFieldsFragment = {
     symbol: 'tBTC',
     decimals: 5,
   },
+  available: '125600000',
+  used: '125600000',
+  deposited: '125600000',
 };
 const singleRowData = [singleRow];
 
 describe('AccountsTable', () => {
   it('should render successfully', async () => {
-    await act(async () => {
-      const { baseElement } = render(<AccountsTable data={[]} />);
-      expect(baseElement).toBeTruthy();
-    });
+    const { baseElement } = render(
+      <AccountTable rowData={[]} onClickAsset={() => null} />
+    );
+    expect(baseElement).toBeTruthy();
   });
 
   it('should render correct columns', async () => {
-    act(async () => {
-      render(<AccountsTable data={singleRowData} />);
-      await waitFor(async () => {
-        const headers = await screen.getAllByRole('columnheader');
-        expect(headers).toHaveLength(4);
-        expect(
-          headers.map((h) =>
-            h.querySelector('[ref="eText"]')?.textContent?.trim()
-          )
-        ).toEqual(['Asset', 'Type', 'Market', 'Balance']);
-      });
+    let headers: HTMLElement[];
+    await act(async () => {
+      render(
+        <AccountTable rowData={singleRowData} onClickAsset={() => null} />
+      );
+    });
+    await waitFor(async () => {
```

Comment:
chore (blocking): By wrapping rendering of the grid in act you are allowing the component to finish rendering. This makes `await waitFor` redundant here

The following should suffice
```
it('should render correct columns', async () => {
    await act(async () => {
      render(
        <AccountTable rowData={singleRowData} onClickAsset={() => null} />
      );
    });
    const headers = screen.getAllByRole('columnheader');
    expect(headers).toHaveLength(6);
    expect(
      headers?.map((h) => h.querySelector('[ref="eText"]')?.textContent?.trim())
    ).toEqual(['Asset', 'Deposited', 'Used', '', '', '']);
  });
```

Reply:
updated